module/multimedia/tests/Domain/Entity/MultimediaTest.php

```diff
@@ -96,9 +96,9 @@ public function testAltManipulation(): void
     }
 
     /**
-     * @return AbstractMultimedia
+     * @return Multimedia
      */
-    private function getClass(): AbstractMultimedia
+    private function getClass(): Multimedia
```

Comment:
It is a mechanism for create class from abstract class for test, is not needed here